refactor people and simple center of excellence page

sgratzl opened the pull request:
adds a simple center of excellence page:

![image](https://user-images.githubusercontent.com/4129778/102921400-52184600-445a-11eb-9a4b-85cf76c345dd.png)

In addition, I refactored how people/team members are defined and handled. This unifies the blog authors, team members, and excellence members into one data structure. 

The page doesn't follow the design since the design doesn't consider the new structure that the content document is suggesting.

tildechris commented:
This looks good to me, but let's wait for Raph to take a look.

tildechris commented:
Reviewed this with @RoniRos this morning.  His comments:

Just a few requests:
* Headline/title should change to "National Center of Excellence for Flu Forecasting".
* Add somewhere that it was established in 2019.
* Add prominently near the top that this is a CDC-designated, CDC-funded CoE, and at the bottom start the Thank you's with a prominent thank you to CDC, at higher prominence than the others.
* The UMass reference (very bottom) should not be a "Thank you", but rather simply a shout-out to our sister organization, the UMass Amherst based Center of Excellence for flu forecasting, adn linked to the Reich's lab, https://reichlab.io/.
* The sentence "Over the years we’ve been very lucky to work alongside many accomplished organizations and researchers." should be replaced with something like "We are grateful to the following organizations for their steadfast support of our center".
* We should add a list of publications that were at least partially funded by the center.  May you can have Christy reach out to Baruch, Marc and Wilbert?

I'll follow-up on the last point, but @sgratzl are the other points clear enough to address?

sgratzl commented:
> * Headline/title should change to "National Center of Excellence for Flu Forecasting".

done

> 
> * Add somewhere that it was established in 2019.
> 
> * Add prominently near the top that this is a CDC-designated, CDC-funded CoE, and at the bottom start the Thank you's with a prominent thank you to CDC, at higher prominence than the others.

I updated the first and sentence, please take a look

> 
> * The UMass reference (very bottom) should not be a "Thank you", but rather simply a shout-out to our sister organization, the UMass Amherst based Center of Excellence for flu forecasting, adn linked to the Reich's lab, https://reichlab.io/.
> 

I added a `Valued Sister Organizations` section

> * The sentence "Over the years we’ve been very lucky to work alongside many accomplished organizations and researchers." should be replaced with something like "We are grateful to the following organizations for their steadfast support of our center".
> 

done

> * We should add a list of publications that were at least partially funded by the center.  May you can have Christy reach out to Baruch, Marc and Wilbert?

I prepared the list, in the list of publication -> /about/publications.md if the publication has a flag `coe: true` it will automatically appear on this page, too

tildechris commented:
Thanks @sgratzl.  I'm getting the updated list of publications in order, I'll update here when we have that list.

sgratzl commented:
> @sgratzl can you revert this merge? I met with @RoniRos who is putting together some additional changes. I will bring @Letsgohyde up to speed on the edits that need to be made.

follow up in https://github.com/cmu-delphi/www-main/pull/292

Letsgohyde commented:
hey guys, I updated the design based on Roni's feedback. 
![DELPHI_Center-of-Excel-page_v1 (1)](https://user-images.githubusercontent.com/7318770/115093610-c70f0000-9ecf-11eb-8177-9ab309cfdbcd.png)

https://www.figma.com/file/IiYu74VmEi4GVS4bnYaTd6/Delphi-Site-Redesign?node-id=2843%3A224

Letsgohyde commented:
Here are the 2 logos are the bottom of the page: 
![CDC-logo](https://user-images.githubusercontent.com/7318770/115093811-59170880-9ed0-11eb-99d5-df45cadd96ab.png)
![UMass-logo](https://user-images.githubusercontent.com/7318770/115093812-59af9f00-9ed0-11eb-9f4c-68581fab265c.png)

Letsgohyde commented:
Tablet design: https://www.figma.com/file/IiYu74VmEi4GVS4bnYaTd6/Delphi-Site-Redesign?node-id=2843%3A770
Mobile design: https://www.figma.com/file/IiYu74VmEi4GVS4bnYaTd6/Delphi-Site-Redesign?node-id=2843%3A305

![DELPHI_Center-of-Excel-page_v1 (2)](https://user-images.githubusercontent.com/7318770/115094426-50273680-9ed2-11eb-94f3-7d42675610cf.png)

RoniRos commented:
Thanks Raph!  This looks very nice!

Just a few small tweaks left:

  *   The second paragraph in the “About the Center” box should be eliminated.
  *   The “Special thanks to CDC” at the very bottom should be bigger / more salient than the list of supporters above it.
  *   “Harvard College” should be changed to “Harvard University”
  *   “UHG / Optum” should be changed to “UnitedHealth / Optum”  (note the lack of space between “United” and “Health”)
  *   CDC does not allow the use of its logo.  One way we finessed it is by inventing our own 😉  :  instead of a logo, we used a logo-like square or circle with big bold black-and-white “US CDC” in it.  You may have seen it in an older version of our website.
  *   I don’t know whether U Mass allows the use of their logo.  Regardless, I think it might be more appropriate to use the Forecast Hub<https://covid19forecasthub.org/> logo or the Reich’s lab<https://reichlab.io/> logo instead – I am sure Nick will be fine with that.

Many thanks,

-Roni

From: raphael hyde ***@***.***>
Sent: Friday, April 16, 2021 7:22 PM
To: cmu-delphi/www-main ***@***.***>
Cc: RoniRos ***@***.***>; Mention ***@***.***>
Subject: Re: [cmu-delphi/www-main] refactor people and simple center of excellence page (#193)


hey guys, I updated the design based on Roni's feedback.
<https://user-images.githubusercontent.com/7318770/115093610-c70f0000-9ecf-11eb-8177-9ab309cfdbcd.png>

https://www.figma.com/file/IiYu74VmEi4GVS4bnYaTd6/Delphi-Site-Redesign?node-id=2843%3A224

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cmu-delphi/www-main/pull/193#issuecomment-821720050>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFQ3C3XGCZ7BYJA2AYMFPJ3TJDBALANCNFSM4VF7VYWQ>.

sgratzl commented:
this PR is closed, either comment on the new PR in #292  or the underlying issue at #32

RoniRos commented:
Thank you for the swift feedback @Roni Rosenfeld ***@***.***> - I'll
make those changes and we'll get this implemented :)

On Fri, Apr 16, 2021 at 6:36 PM Roni Rosenfeld ***@***.***> wrote:

> Thanks Raph!  This looks very nice!
>
>
>
> Just a few small tweaks left:
>
>    - The second paragraph in the “About the Center” box should be
>    eliminated.
>    - The “Special thanks to CDC” at the very bottom should be bigger /
>    more salient than the list of supporters above it.
>    - “Harvard College” should be changed to “Harvard University”
>    - “UHG / Optum” should be changed to “UnitedHealth / Optum”  (note the
>    lack of space between “United” and “Health”)
>    - CDC does not allow the use of its logo.  One way we finessed it is
>    by inventing our own 😉  :  instead of a logo, we used a logo-like
>    square or circle with big bold black-and-white “US CDC” in it.  You may
>    have seen it in an older version of our website.
>    - I don’t know whether U Mass allows the use of their logo.
>    Regardless, I think it might be more appropriate to use the Forecast
>    Hub <https://covid19forecasthub.org/> logo or the Reich’s lab
>    <https://reichlab.io/> logo instead – I am sure Nick will be fine with
>    that.
>
>
>
> Many thanks,
>
>
>
> -Roni
>
>
>
> *From:* raphael hyde ***@***.***>
> *Sent:* Friday, April 16, 2021 7:22 PM
> *To:* cmu-delphi/www-main ***@***.***>
> *Cc:* RoniRos ***@***.***>; Mention <
> ***@***.***>
> *Subject:* Re: [cmu-delphi/www-main] refactor people and simple center of
> excellence page (#193)
>
>
>
> hey guys, I updated the design based on Roni's feedback.
> [image: DELPHI_Center-of-Excel-page_v1 (1)]
> <https://user-images.githubusercontent.com/7318770/115093610-c70f0000-9ecf-11eb-8177-9ab309cfdbcd.png>
>
>
> https://www.figma.com/file/IiYu74VmEi4GVS4bnYaTd6/Delphi-Site-Redesign?node-id=2843%3A224
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cmu-delphi/www-main/pull/193#issuecomment-821720050>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFQ3C3XGCZ7BYJA2AYMFPJ3TJDBALANCNFSM4VF7VYWQ>
> .
>


-- 

Raphael Hyde

***@***.***

Interaction Designer
